Dockerfile

```diff
@@ -1,11 +1,25 @@
-FROM python:3.11-slim-buster
+FROM python:slim
+
+LABEL org.opencontainers.image.source="https://github.com/toddrob99/searcharr"
+LABEL org.opencontainers.image.description="Docker for SEARCHARR"
+LABEL Name=Searcharr Version="v1.2-beta"
+
+ARG TARGETPLATFORM BUILDPLATFORM
 
-LABEL Name=Searcharr Version=1.2
 
 WORKDIR /app
+
+RUN chmod -R 777 /app && \
+    chmod -R +x /app && \
+    chmod -R 705 /app
+
 ADD . /app
 
-RUN python3 -m pip install --upgrade pip
-RUN python3 -m pip install -r requirements.txt
+RUN apt-get update -y && \
+    apt-get upgrade -y
```

Comment:
Why bother updating--are any apt packages even being used?

Reply:
no but keeping apt packages uptodate is a good practice.